tests/run-make/no-cdylib-as-rdylib/rmake.rs

```diff
@@ -0,0 +1,16 @@
+// This test produces an rlib and a cdylib from bar.rs.
+// Then, foo.rs attempts to link to the bar library.
+// If the test works correctly, rustc will favour the rlib and ignore the cdylib.
```

Comment:
It's more accurate to say "if the test passes, rustc favored the rlib and ignored the cdylib".